.github/workflows/test.yml

```diff
@@ -27,8 +27,12 @@ jobs:
       POSTGRES_DB: sims
       POSTGRES_PORT: 5432
       POSTGRES_USER: admin
+      REDIS_PORT: 6379
+      REDIS_PASSWORD: redis
+      REDIS_STANDALONE_MODE: "true"
       API_PORT: 3000
       APP_PORT: 8080
+      QUEUES_PORT: 3001
```

Comment:
I can see this ENV variable declared but it does not seem to be consumed, it is hard-coded also in the `main.ts`. Am I missing something?

Reply:
Yes it is. I will change it.